sys/syz-extract/linux.go

```diff
@@ -80,11 +81,18 @@ func (*linux) prepareArch(arch *Arch) error {
 	}
 	kernelDir := arch.sourceDir
 	makeArgs := build.LinuxMakeArgs(arch.target, "", "", "", arch.buildDir)
-	out, err := osutil.RunCmd(time.Hour, kernelDir, "make", append(makeArgs, "defconfig")...)
-	if err != nil {
-		return fmt.Errorf("make defconfig failed: %w\n%s", err, out)
+	if arch.configFile != "" {
+		err := copyFile(arch.configFile, arch.buildDir+"/.config")
```

Comment:
use `filepath.Join(arch.buildDir, ".config")`
It correctly adds separators and knows what's the actual separator on the OS (/ or \)